README.md

```diff
@@ -162,6 +162,20 @@ The following work on normal and visual modes:
 
 -   `:Tocv`: Same as `:Toc` for symmetry with `:Toch` and `Tocv`.
 
+## Issue reporting
+
+Issues are tracked within GitHub.
+
+When reporting issues, your report is more effective if you include a minimal example file that reproduces the problem. Try to trim out as much as possible, until you have the smallest possible file that still reproduces the issue. Paste the example inline into your issue report, quoted using four spaces at the beginning of each line, like this example from issue [#189](https://github.com/plasticboy/vim-markdown/issues/189):
+
+```
+Minimal example:
+
+    ```
+    =
+    ```
+    bad!
+```
```

Comment:
For a user filing a bug report, it's not necessarily obvious how to quote the markdown to achieve the effect you did in #189. It took me a few minutes to figure it out just to make that example. I included the extra detail and the example to save users the trouble of discovering the right way to do it on their own. Shortening the text and removing the example would remove this detail, and make it less likely that a typical bug reporter is going to bother to include an example in a format convenient for you. Since you probably prefer clear bug reports, and want to save your users time, I'd recommend leaving it in there.